src/lightning/fabric/cli.py

```diff
@@ -44,7 +48,33 @@ def _get_supported_strategies() -> List[str]:
 if _CLICK_AVAILABLE:
     import click
 
-    @click.command(
+    def _legacy_main() -> None:
+        """Legacy CLI handler for fabric.
+
+        Raises deprecation warning and runs through fabric cli if necessary, else runs the entrypoint directly
+
+        """
+        warnings.warn("lightning run model is deprecated and will be removed in future versions."
+            " Please call `fabric run model` instead.", DeprecationWarning)
+        args = sys.argv[1:]
+        if args and args[0] == "run" and args[1] == "model":
+            # raise Deprecation Warning
```

Comment:
is the comment misplaced?

Reply:
that was a todo I forgot to remove. Done :)